staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package bootstrap provides a token authenticator for TLS bootstrap secrets.
+*/
+package bootstrap
+
+import (
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
I was trying to mimic the service account authenticator[0] and that uses the API server's internal storage directly[1]. That uses the API server package[2].

I planned to hook it up the same way we do the service account authenticator, and that's why it's this way. Is the service account authenticator using old practices?

[0] https://github.com/kubernetes/kubernetes/blob/8621bd3e50d211446fcc3fd781d6c2637b87b01c/pkg/kubeapiserver/authenticator/config.go#L69
[1] https://github.com/kubernetes/kubernetes/blob/8621bd3e50d211446fcc3fd781d6c2637b87b01c/cmd/kube-apiserver/app/server.go#L252
[2] https://github.com/kubernetes/kubernetes/blob/8621bd3e50d211446fcc3fd781d6c2637b87b01c/pkg/serviceaccount/jwt.go#L32